<launch>
  <arg name="map_file" default="$(find bwi_exp1)/maps/exp1.yaml" />
  <arg name="graph_file" default="$(find bwi_exp1)/maps/graph.yaml" />
  <node name="$(anon view_graph)" pkg="topological_mapper" type="view_graph"
    args="$(arg map_file) $(arg graph_file)" />
</launch>
